app/views/includes/header.scala.html

```diff
@@ -9,4 +9,15 @@
     <link rel='stylesheet' type='text/css' href='@routes.Assets.at("/css/font-awesome.min.css")'/>
     <link rel='stylesheet' type='text/css' href='@routes.Assets.at("/css/style.css")'/>
     <link rel='stylesheet' type='text/css' href='@routes.Assets.at("/css/carbon.css")'/>
+
+    <!-- Social bits -->
+    <meta property='og:type' content='website' />
+    <meta property='og:url' content='https://sdkman.io/' />
```

Comment:
Ideally, this should probably be the absolute URL to the current page, is there a `@` variable for this?

If not, it's not the end of the world 🤷

Reply:
No, but we should be able to compose this. I'll see if I can get it to work on the weekend.

Reply:
@bdemers I've managed to propagate this through the request and will add a commit to compose the URL.